### PR TITLE
SCTP echo client/server multi-homing examples

### PR DESCRIPTION
**Pull request by jestan**
Motivation:
There are no Netty SCTP examples on multi-homing.

Modifications:
- Added new example classes based on echo client/server example

Result:
Better documentation

### CODE REVIEW

**Comment by netkins** [comment]
**AUTOMATED MESSAGE for ADMINS**:<br>Please verify and accept the pull request.<br>To accept, say `@netkins accept`<br>To build again, say `@netkins build`<br>To whitelist the author, say `@netkins whitelist`

**Comment by normanmaurer** [comment]
@netkins accept

File: example/src/main/java/io/netty/example/sctp/multihoming/SctpMultiHomingEchoClient.java [hunk 1]
```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright 2012 The Netty Project
```
Comment (normanmaurer) [inline example/src/main/java/io/netty/example/sctp/multihoming/SctpMultiHomingEchoClient.java]: 2015

**Comment by normanmaurer** [comment]
All package-private fields should be private

File: example/src/main/java/io/netty/example/sctp/multihoming/SctpMultiHomingEchoClient.java [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+
+import java.net.InetAddress;
+import java.net.InetSocketAddress;
+
+/**
+ * SCTP Echo Client with multi-homing support.
+ */
+public final class SctpMultiHomingEchoClient {
+
+    static final String CLIENT_PRIMARY_HOST = System.getProperty("host.primary", "127.0.0.1");
```
Comment (normanmaurer) [inline example/src/main/java/io/netty/example/sctp/multihoming/SctpMultiHomingEchoClient.java]: Make these fields private

File: example/src/main/java/io/netty/example/sctp/multihoming/SctpMultiHomingEchoServer.java [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+
+/**
+ * SCTP Echo Server with multi-homing support.
+ */
+public final class SctpMultiHomingEchoServer {
+
+    static final String SERVER_PRIMARY_HOST = System.getProperty("host.primary", "127.0.0.1");
+    static final String SERVER_SECONDARY_HOST = System.getProperty("host.secondary", "127.0.0.2");
+
+    static final int SERVER_PORT = Integer.parseInt(System.getProperty("port", "8007"));
```
Comment (normanmaurer) [inline example/src/main/java/io/netty/example/sctp/multihoming/SctpMultiHomingEchoServer.java]: make these fields private

File: example/src/main/java/io/netty/example/sctp/multihoming/SctpMultiHomingEchoServer.java [hunk 4]
```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright 2012 The Netty Project
```
Comment (normanmaurer) [inline example/src/main/java/io/netty/example/sctp/multihoming/SctpMultiHomingEchoServer.java]: 2015

**Comment by normanmaurer** [comment]
@jestan please address my comments.

**Comment by jestan** [comment]
@normanmaurer  sure, I will update and let you know

**Comment by normanmaurer** [comment]
@jestan thanks!

**Comment by normanmaurer** [comment]
@jestan ping

**Comment by jestan** [comment]
@normanmaurer done :).

**Comment by normanmaurer** [comment]
@jestan thanks! Now please squash into one commit and we are done :)

**Comment by normanmaurer** [comment]
@jestan nevermind...squashed for you and cherry-picked into 4.0 (74f67ddca77849d0d61cc795686d16683d054923), 4.1 (4877fffa0a591bdb6fe21af6192986b74102d17d) and master (6fe13260bf5671eb89d356f0b94c01bdd54efb5f)

**Pull request closed without merging**